scripts/scil_compute_MT_maps.py

```diff
@@ -0,0 +1,377 @@
+reference.
+
+The output consist in two types of images:
+        Three contrasts images : MT-off, MT-on and T1weighted images.
+        MT maps corrected or not for an empiric B1 correction maps.
+
+Input Data recommendation:
+  - it is recommended to use dcm2niix (v1.0.20200331) to convert data
+    https://github.com/rordenlab/dcm2niix/releases/tag/v1.0.20200331
+  - dcm2niix conversion will create all echo files for each contrast and
+    corresponding json files
+  - all input must have a matching json file with the same filename
+  - all contrasts must have a same number of echoes and coregistered
+    between them before running the script.
+  - Mask must be coregistered to the echo images
+  - ANTs can be used for the registration steps (http://stnava.github.io/ANTs/)
+
+
+The output consist in two types of images in two folders :
+  1. Contrasts_MT_maps which contains the 2 contrast images
+      - MT-off.nii.gz : pulses applied at positive frequency
+      - MT-on.nii.gz : pulses applied at negative frequency
+      - T1w.nii.gz : anatomical T1 reference images
+
+
+  2. MT_native_maps which contains the 4 myelin maps
+      - MTR.nii.gz : Magnetization Transfer Ratio map
+      The MT ratio is a measure reflecting the amount of bound protons.
+
+      - MTsat.nii.gz : Magnetization Transfer saturation map
+      The MT saturation is a pseudo-quantitative maps representing
+      the signal change between the bound and free water pools.
+
+>>> scil_compute_ihMT_maps.py path/to/output/directory path/to/mask_bin.nii.gz
+    --in_mtoff path/to/echo*mtoff.nii.gz --in_mton path/to/echo*mton.nii.gz
+    --in_t1w path/to/echo*T1w.nii.gz
+
+"""
+
+import argparse
+import os
+import json
+
+import nibabel as nib
+import numpy as np
+import scipy.ndimage
+
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_output_dirs_exist_and_empty)
+
+EPILOG = """
+Helms G, Dathe H, Kallenberg K, Dechent P. High-resolution maps of
+magnetization transfer with inherent correction for RF inhomogeneity
+and T1 relaxation obtained from 3D FLASH MRI. Magnetic Resonance in Medicine.
+2008;60(6):1396-407.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('out_dir',
+                   help='Path to output folder.')
+    p.add_argument('in_mask',
+                   help='Path to the T1 binary brain mask. Must be the sum '
+                        'of the three tissue probability maps from '
+                        'T1 segmentation (GM+WM+CSF).')
+    p.add_argument('--out_prefix',
+                   help='Prefix to be used for each output image.')
+    p.add_argument('--in_B1_map', action='store_true',
+                   help='Path to B1 coregister map to MT contrasts.')
```

Comment:
this option does not work. Please correct. If I remove the action='store_true', it works.